tests/test_model_evaluation.py

```diff
@@ -1,90 +1,76 @@
-import os
-
 import networkx
 import pytest
 import sympy
 
-from cellmlmanip import load_model
-
 
-class TestModelEvaluation():
+def test_model_evaluation(hh_model):
```

Comment:
This could probably go in somewhere else then? Or maybe leave for a bigger test reorganisation :-)